notification/app/notification/NotificationApplicationLoader.scala

```diff
@@ -10,11 +11,15 @@ import azure.NotificationHubClient
 import com.softwaremill.macwire._
 import controllers.Main
 import _root_.models.NewsstandShardConfig
+import com.google.auth.oauth2.GoogleCredentials
+import com.google.firebase.messaging.{AndroidConfig, FirebaseMessaging}
```

Comment:
A few imports appear to be flagged as unused in Intellij, including AndroidConfig

Reply:
Done